### DISCUSSION TITLE
Bug: Win 11 graphics issue

### DISCUSSION THREAD

**Discussion by quickfold** [freeplane/freeplane#1965] [Help] [answered]
I'm using FP 1.11.14 and Win 11 with updated graphics drivers. The same FP version on the same computer and graphics card worked fine on Win 10. After Win 11 upgrade, I get a graphics bug that makes FP almost impossible to use. Let me know how I can help troubleshoot.

https://github.com/user-attachments/assets/fb2845f0-d172-4152-9bf9-1bed820da918

**Comment by dpolivaev** [comment]
It is an old problem with no verified solution, see for example https://github.com/freeplane/freeplane/issues/230

I have found interesting documentation about possible fixes. Please check it and say if any of the suggested options helps:

https://docs.oracle.com/en/java/javase/22/troubleshoot/java-2d-pipeline-rendering-and-properties.html#GUID-AAB8CC86-652B-4A78-83EB-CDC3F5677A48

If not, you can look at the other pages of this document.

You can add the options to  freeplane.l4j.ini and  freeplaneConsole.l4j.ini, they are used by the exe launchers.

**Reply by dpolivaev** [reply]
(you can replace javase/22 in the link by any other version e.g. 15, 17, 21)

**Comment by quickfold** [comment, marked as answer]
Thanks, I disabled DirectDraw in those files with `-Dsun.java2d.noddraw=true` and it appears to have fixed things. I'll report back if the bug reappears.

**Comment by quickfold** [comment]
Those files seem to be overwritten on every update, so I need to manually add that command into both files. Is there any way to avoid having to do this each time?

**Reply by dpolivaev** [reply]
Preview 1.12.5_01
